iOS Example Frame/iOS Example Frame/Factory/Factory+UITest.swift

```diff
@@ -28,6 +28,22 @@ extension Factory {
 
         return viewController
     }
+
```

Comment:
please move `iOS Example Frame/iOS Example Frame/Factory/Factory+UITest.swift` to UITest target

Reply:
The call site for the API should be able to discover the declarations of that API.

In practice that means if we were to declare the methods contained within this file to UITest target, our application would need to have access to the UITest module. That would be a wrong dependency, as UITest needs to depend on source code it tests, not the other way around.